skopt/dummy_opt.py

```diff
@@ -43,22 +44,22 @@ def dummy_minimize(func, bounds=None, maxiter=1000, random_state=None):
     rng = check_random_state(random_state)
 
     # Bounds
-    num_params = len(bounds)
-    lower_bounds, upper_bounds = extract_bounds(bounds)
-    diff = upper_bounds - lower_bounds
+    n_params = len(bounds)
+    lb, ub = extract_bounds(bounds)
 
     # Sample each parameter uniformly between 0 and 1 and then rescale.
```

Comment:
That isn't true anymore, we sample some uniform numbers and then multiply. To me this comment makes it sound like we do everything in the domain [0,1] and only at the very every end scale back to the original domain.